### ISSUE TITLE
[eFEAT] Bundle JRE in Executable

### ISSUE THREAD

**Issue by pierpaolocira** [akasolace/HO#202]
This request follows #198 

When #106 is done, verify if the MacOS `.app`  can be distributed by embedding the JRE.

From the related issue

> It is not clear if it is related either to the code (not yet compatible with Java 11 (see issue #106 ) or to an issue in the plugin used to build the app (edu.sc.seis.macAppBundle, see crotwell/gradle-macappbundle#56 and crotwell/gradle-macappbundle#49 )... or perhaps both.

**Comment by akasolace** [comment]
@tychobrailleur @wsbrenk @yaute74  do you think this is something we want to do ?

**Comment by pierpaolocira** [comment]
Hello... I did something alike for my company few years ago, because it was in line with Oracle guidelines.
This was just a .EXE (with a budle of libraries in subfolders) to be launched without the need to install/configure any JRE (and, even better, without generating conflicts with any already existing JRE on the host machine).

Accordingly to Oracle and what I have read (and experienced), in my personal opinion, this should be the way to go: the user download and use it... without caring of the JRE versione used to build, which should change more frequently, since Oracle has introduced the new release plan.

If I remember well, this feature was removed in the MacOS APP because the gradle plugin used to build for MacOS wasn't supporting JRE different than version 8 at that time (and was generating an invalid APP without triggering any exception).

But ok... just my opinione...
